dnscommon.r2py

```diff
@@ -519,9 +520,25 @@ def _read_single_answer(answer_index, dns_query_data):
 
   # There are a lot of different types of queries. We can parse all of these ones 
   # in the same way, though.
-  simple_answers = ['A', 'NS', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR', 'MX', 'PTR']
+  simple_answers = ['A', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR']
   if answer_type in simple_answers:
     read_index, resource_data['address'] = _parse_answer_address(read_index, dns_query_data)
+  elif answer_type == 'NS':
+    read_index, resource_data['nameserver'] = _parse_address(read_index,dns_query_data)
+  elif answer_type == 'PTR':
+    read_index, resource_data['domain'] = _parse_address(read_index, dns_query_data)
+  elif answer_type == 'MX':
+    read_index+=2
```

Comment:
````
;; QUESTION SECTION:
;google.com.			IN	MX

;; ANSWER SECTION:
google.com.		301	IN	MX	40 alt3.aspmx.l.google.com.
google.com.		301	IN	MX	30 alt2.aspmx.l.google.com.
google.com.		301	IN	MX	50 alt4.aspmx.l.google.com.
google.com.		301	IN	MX	20 alt1.aspmx.l.google.com.
google.com.		301	IN	MX	10 aspmx.l.google.com.
````
As we can see there is some values like 10,20,30... before the MX records, we need to increase index to get to correct data from raw packet.